doc/api/buffer.md

```diff
@@ -1221,12 +1221,12 @@ changes:
     description: The `encoding` parameter is supported now.
 -->
 
-* `value` {string|Buffer|integer} The value to fill `buf` with
-* `offset` {integer} Where to start filling `buf`. **Default:** `0`
+* `value` {string|Buffer|integer} The value to fill `buf` with.
+* `offset` {integer} Number of bytes in at which to start filling `buf`. **Default:** `0`
```

Comment:
Can we rephrase this again? I do not feel this is any clearer than before.

Something like `Number of bytes from which on the buffer will be filled`? The same applies to all other occurrences of that sentence / change.

Reply:
How about `Number of bytes to skip before filling buffer.`?

Reply:
Yes, that will also work.

Reply:
Done